### ISSUE TITLE
Where to find a changelog? 

### ISSUE THREAD

**Issue by meznaric** [callstack/haul#750]
I haven't upgraded haul on my project for quite some time, now all my `@haul-bundler/*` packages are all outdated. How should I approach upgrading?

I noticed one of the patches was merged that I need but I have no idea if it was released.

It's not the first time I encountered this. Simply having changelog would make this much easier.

**Comment by zamotany** [comment]
I agree, there's much to be done in terms of maintenance, changelog etc. For now, the best source of truth is to look a commit history: https://github.com/callstack/haul/commits/master
There are `chore: publish` commits which contain published packages and their versions, so when you find a patch/commit you're interested, check if it has `chore: publish` commit above it.

**Comment by meznaric** [comment]
Thanks for swift response! I see how it works now, basically versions don't necessarily need to match, they are released/upgraded as needed. Commits in between will tell me what the change was.

I see you put a "good first issue" tag to it, what do you imagine as part of this?
I can see 2 things being done:
 - Just documentation improved that explain what you explained to me in more detail
 - Automated changelog generation. If I was to build shell/node script that would generate/update CHANGELOG.md based on the commits, would you integrate it with CI? - I haven no experience with circle.

**Comment by zamotany** [comment]
The process of generating changelog can be automated by Learn, which we use for managing monorepo, so it boils down to add proper options to `lerna` commands to generate it since the commit messages follow Conventional Commits: https://github.com/lerna/lerna/tree/master/commands/version#--conventional-commits
The cherry on top would be also a automatic Github Releases creation, but that's more complex so can be left for later.
